src/web_ui/__init__.py

```diff
@@ -80,6 +92,26 @@ def env_scan():
     )
 
 
+@app.route("/realtime-readings")
+def realtime_readings():
+    devices = []
+    DATA_TOPIC = f'a/{node_id}/data'
```

Comment:
Not entirely sure where this comes from but it's not what you want here (evidently, since it's unused :) ). This is the (legacy) topic on the "main" broker in the cloud, which we are not connecting to here.

In case you want to enforce uniformity across modules (seems like a good idea), you can simply import `MQTT_DATA_TOPIC` from [here](https://github.com/ammpio/ammp-edge/blob/main/src/data_mgmt/helpers/mqtt_pub.py)